tests/FakeItEasy.Analyzer.Tests/Helpers/CodeFixVerifier.cs

```diff
@@ -46,7 +46,6 @@ protected virtual CodeFixProvider GetBasicCodeFixProvider()
         /// <param name="newSource">A class in the form of a string after the code fix was applied to it.</param>
         /// <param name="codeFixIndex">Index determining which code fix to apply if there are multiple.</param>
         /// <param name="allowNewCompilerDiagnostics">A boolean controlling whether or not the test will fail if the code fix introduces other warnings after being applied.</param>
-        [SuppressMessage("Microsoft.Design", "CA1026:DefaultParametersShouldNotBeUsed", Justification = "It's not intended to be a public API, so it doesn't matter")]

```

Comment:
I disabled this warning in the ruleset for tests, since it only matters for public APIs